packages/datagateway-dataview/src/views/landing/isis/isisInvestigationLanding.component.tsx

```diff
@@ -354,6 +354,13 @@ const LandingPage = (props: LandingPageProps): React.ReactElement => {
                 ? data[0].summary
                 : 'Description not provided'}
             </Typography>
+            {formattedUsers.length > 0 ? (
+              ''
+            ) : (
+              <Typography data-testid="investigation-details-panel-no-name">
+                {t('investigations.details.users.no_name')}
+              </Typography>
+            )}
```

Comment:
This should be placed with the heading below, currently when there are no users, the users heading doesn't display so either this isn't needed at all or the below code should be modified to always show the heading.
Currently it gives:
![image](https://user-images.githubusercontent.com/90245114/151548474-bedd7649-a756-46ec-b41a-835dd927d71d.png) at /browseStudyHierarchy/instrument/32/study/90277599/investigation/24081034 using prod data.